pkg/storkctl/actionstatus.go

```diff
@@ -0,0 +1,156 @@
+					return
+				}
+			}
+
+			// filter it down to actions with actionType failover
+			failoverActionList = new(storkv1.ActionList)
+			for _, action := range actions.Items {
+				if action.Spec.ActionType == storkv1.ActionTypeFailover {
+					failoverActionList.Items = append(failoverActionList.Items, action)
+				}
+			}
+
+			if len(failoverActionList.Items) == 0 {
+				handleEmptyList(ioStreams.Out)
+				return
+			}
+
+			if err := printObjects(c, failoverActionList, cmdFactory, drActionColumns, drActionPrinter, ioStreams.Out); err != nil {
+				util.CheckErr(err)
+				return
+			}
+		},
+	}
+	cmdFactory.BindGetFlags(getFailoverCommand.Flags())
+	return getFailoverCommand
+}
+
+func drActionPrinter(actionList *storkv1.ActionList, options printers.GenerateOptions) ([]metav1beta1.TableRow, error) {
+	if actionList == nil {
+		return nil, nil
+	}
+
+	rows := make([]metav1beta1.TableRow, 0)
+	for _, action := range actionList.Items {
+		var row metav1beta1.TableRow
+		creationTime := toTimeString(action.CreationTimestamp.Time)
+		additionalInfo := action.Status.Reason
+		if action.Status.Summary != nil {
+			if len(action.Status.Summary.FailoverSummaryItem) > 0 {
+				// find out number of successfully scaled up namespaces
+				totalNamespaces := len(action.Status.Summary.FailoverSummaryItem)
+				successfulNamespaces := 0
+				rollbackSuccessfulNamespaces := 0
+				rollbackFailedNamespaces := 0
+				for _, item := range action.Status.Summary.FailoverSummaryItem {
+					if item.Status == storkv1.ActionStatusSuccessful {
+						successfulNamespaces++
+					} else if item.Status == storkv1.ActionStatusRollbackSuccessful {
+						rollbackSuccessfulNamespaces++
+					} else if item.Status == storkv1.ActionStatusRollbackFailed {
+						rollbackFailedNamespaces++
+					}
+				}
+				if rollbackSuccessfulNamespaces > 0 || rollbackFailedNamespaces > 0 {
+					additionalInfo = fmt.Sprintf("Rolled back Apps in : %d/%d namespaces", rollbackSuccessfulNamespaces, totalNamespaces)
+					// In case of rollback we have a failure reason as well
+					additionalInfo += " ; " + action.Status.Reason
+				} else {
+					additionalInfo = fmt.Sprintf("Scaled up Apps in : %d/%d namespaces", successfulNamespaces, totalNamespaces)
+				}
+			}
+		}
+		row = getRow(&action,
+			[]interface{}{action.Name,
+				creationTime,
+				prettyDRStageNames(action.Status.Stage, action.Spec.ActionType),
+				action.Status.Status,
+				additionalInfo,
+			},
+		)
+		rows = append(rows, row)
+	}
+	return rows, nil
+}
+
+func prettyDRStageNames(stage storkv1.ActionStageType, actionType storkv1.ActionType) string {
+	switch stage {
+	case storkv1.ActionStageInitial:
+		return "Validations"
+	case storkv1.ActionStageScaleDownSource:
+		return "Scale Down (on source)"
+	case storkv1.ActionStageScaleDownDestination:
+		return "Scale Down (on destination)"
+	case storkv1.ActionStageWaitAfterScaleDown:
+		if actionType == storkv1.ActionTypeFailover {
+			return "Scale Down (on source)"
```

Comment:
Let's make it explicit: Waiting for Apps to Scale Down (on source)

Reply:
okay will address